### DISCUSSION TITLE
No response from GitHub team from for Student Developer PAck

### DISCUSSION THREAD

**Discussion by yash-278** [community/community#21818] [New to GitHub]
<p>Hi,</p>
<p>I have applied for Student developer pack on 20 Jan of 2020 ,the status is still pending. I don’t have the school issued email id, so applied with the school document, with the valid date written on it.</p>
<p>It says that Support will contact in 10 days , but it’s almost 15+ days.</p>
<p>Kindly Help.</p>
<p>Thanks</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/yash-278">@yash-278</a>,</p>
<p>Thanks for reaching out! It takes a few weeks for the review and approval of the pack. Thanks for your patience</p>

**Comment by Deeprajdey** [comment]
<p>       </p>
<p>My request is still pending. I have applied it on Feb 21, 2020. I have also mailed to the support team and got a reply from the team and still waiting for the acceptance of the developer pack request…  </p>
